### ISSUE TITLE
Response Time to get a metadata

### ISSUE THREAD

**Issue by mclaudon-mto** [OpenWIS/openwis#238]
For some reason, it takes roughly 10 secondes to get the response for a request to a metadata. 
It seems to be generic in all GISCs. For example  : 
MeteoFrance : http://wispi.meteo.fr/openwis-user-portal/srv/en/iso19139.xml?uuid=urn:x-wmo:md:fr.meteofrance::MF-NWP-HIGHRES-AROME-0025-FRANCE-WMS
Bom : http://wis.bom.gov.au/openwis-user-portal/srv/en/iso19139.xml?id=341685 

Is it linked to SOLR ? to the table index in the database ?

At MeteoFrance, it prevents us from validating INSPIRE metadata, because the INSPIRE validator has a short timeout.

**Comment by woollattd** [comment]
do you know if this was the case before the uplift or since you had v3.14.6 installed?

**Comment by mclaudon-mto** [comment]
Yes, for the link I mentioned above, it is a 3.13.1 version of openwis which is used (and still currently operational in Meteo-France).

**Comment by woollattd** [comment]
BOM have 3.13.1 installed on Production too. (I don't think they need to comply with INSPIRE though)
Do you see the same issue on our GISC (v3.14.5)? at https://gisc.metoffice.gov.uk

I just get errors trying to replicate the problem :-( so must be doing something wrong trying to create the address with our metadata

eg
https://gisc.metoffice.gov.uk/openwis-user-portal/srv/en/iso19139.xml?uuid=urn:x-wmo:bulletin:int.wmo.wis::HUMB98EGRR

**Comment by mclaudon-mto** [comment]
The url you mentioned above is wrong (Metadata not found). 

I try with this one on your GISC : 
https://gisc.metoffice.gov.uk/openwis-user-portal/srv/en/iso19139.xml?id=155310

It is a bit faster : 5 seconds.

**Comment by woollattd** [comment]
is 5 seconds enough?
there might have been some improvements in the uplift that have led to the slightly faster time. I am guessing though.

**Comment by mclaudon-mto** [comment]
I am going to check with the team.

**Comment by lmika-bom** [comment]
Some incidental stats from GISC Melbourne: it takes roughly 10 - 30 seconds to get metadata XML using this endpoint (tested from Japan so network might have _some_ impact, but it always took several seconds).

We've done no profiling so I don't know where the slow-down is.  The uplift might make it a little faster, but I imagine the slow-down is somewhere in the application.  If we're are interested in speeding this up (note the "if"), this should be the first thing we do: it could be that this is fixed in GN 3, and therefore version 4.

**Comment by lmika-bom** [comment]
Set to fix for v3 only.  There's issues with the harvester within OpenWIS 4 but this is in a separate ticket.

**Comment by jude2018** [comment]
Per PMC in Helsinki - Météo France to test with KMA or BoM @benjaminsaclier

**Comment by JeyoungRyu** [comment]
As a result of testing in GISC Seoul (v3.14.8), it takes roughly 15 - 20 seconds to get a metadata XML. 
e.g.   http://gisc.kma.go.kr/openwis-user-portal/srv/en/iso19139.xml?id=1363194
